### ISSUE TITLE
Missing Google Managed Service Account for scheduled_queries Module in Terraform

### ISSUE THREAD

**Issue by El7ammadiiiii** [terraform-google-modules/terraform-google-bigquery#301]
### TL;DR

There is an issue with the scheduled_queries module where it assumes the presence of a Google Managed Service Account, but an IAM policy setup error occurs

### Expected behavior

The scheduled_queries module is expected to work without issues according to the specified configuration.

### Observed behavior

When running the scheduled_queries module, an IAM policy error occurs, indicating the absence of a Google Managed Service Account.

### Terraform Configuration

```hcl
[Insert here the Terraform configuration reproducing the issue]
```

### Terraform Version

```sh
[Insert here the output of `terraform version`]
```

### Additional information

It appears that the module assumes the existence of a Google Managed Service Account, which is only created upon enabling the Data Transfer API in the project. Please incorporate this information into the README as a prerequisite. Additionally, confirm that contributions to this repository should adhere to its contributing guidelines and security policy.
